service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/Query.java

```diff
@@ -260,12 +258,14 @@ public QueryResponse sendQueryRequest(IotHubConnectionString iotHubConnectionStr
      * @throws IOException If any of the input parameters are not valid.
      * @throws IotHubException If HTTP response other then status ok is received.
      */
-    public QueryResponse sendQueryRequest(IotHubConnectionString iotHubConnectionString,
-                                          URL url,
-                                          HttpMethod method,
-                                          int httpConnectTimeout,
-                                          int httpReadTimeout,
-                                          Proxy proxy) throws IOException, IotHubException
+    public QueryResponse sendQueryRequest(
+            IotHubConnectionString iotHubConnectionString,
+            URL url,
+            HttpMethod method,
+            int httpConnectTimeout,
+            int httpReadTimeout,
+            Proxy proxy)
+            throws IOException, IotHubException
```

Comment:
Why IOException in non deprecated method?

Reply:
This method sends an HTTP request, which can throw an IOException

Reply:
I see. I missed that the method in line 170 was already deprecated and this was not to remove the IOException.